### PR TITLE
fix: use wildcard values as names

### LINKED ISSUE #148: Run template commands
Hello.

I think there are some mistakes in the DOCS, because this stuff has never worked for me.

```
concurrently "npm:watch-*"

# Equivalent to:
concurrently -n js,css,node "npm run watch-js" "npm run watch-css" "npm run watch-node"

# Any name provided for the wildcard command will be used as a prefix to the wildcard
# part of the script name:
concurrently -n w: npm:watch-*

# Equivalent to:
concurrently -n w:js,w:css,w:node "npm run watch-js" "npm run watch-css" "npm run watch-node"
```

Thx.

### PR DESCRIPTION
**Pull request by mastilver**
fix #148

### CODE REVIEW

File: src/command-parser/expand-npm-wildcard.js [hunk 1]
```diff
@@ -26,9 +26,9 @@ module.exports = class ExpandNpmWildcard {
 
         return this.scripts
             .filter(script => wildcardRegex.test(script))
-            .map(script => Object.assign({}, commandInfo, {
+            .map(script => Object.assign({}, {
                 command: `npm run ${script}${args}`,
-                name: script
+                name: (commandInfo.names || '') +  script.match(wildcardRegex)[1]
```
Comment (gustavohenke) [inline src/command-parser/expand-npm-wildcard.js]: The right property is `name`

File: src/command-parser/expand-npm-wildcard.spec.js [hunk 2]
```diff
@@ -52,3 +52,17 @@ it('caches scripts upon calls', () => {
...
+            'watch-foo': '',
+            'watch-bar': '',
+        }
+    });
+
+    expect(parser.parse({ names: 'w:', command: 'npm run watch-*' })).toEqual([
+        { name: 'w:foo', command: 'npm run watch-foo' },
+        { name: 'w:bar', command: 'npm run watch-bar' },
+    ]);
+})
```
Comment (gustavohenke) [inline src/command-parser/expand-npm-wildcard.spec.js]: missing semicolon

File: src/command-parser/expand-npm-wildcard.spec.js [hunk 3]
```diff
@@ -52,3 +52,17 @@ it('caches scripts upon calls', () => {
...
+
+it('suffix name with wildcard values', () => {
+    readPkg.mockReturnValue({
+        scripts: {
+            'watch-foo': '',
+            'watch-bar': '',
+        }
+    });
+
+    expect(parser.parse({ names: 'w:', command: 'npm run watch-*' })).toEqual([
```
Comment (gustavohenke) [inline src/command-parser/expand-npm-wildcard.spec.js]: will have to fix the property here too to `name` (see above)

**Comment by mastilver** [comment]
My bad, should be all fixed now :)

File: src/command-parser/expand-npm-wildcard.js [hunk 4]
```diff
@@ -28,7 +28,7 @@ module.exports = class ExpandNpmWildcard {
             .filter(script => wildcardRegex.test(script))
             .map(script => Object.assign({}, commandInfo, {
```
Comment (gustavohenke) [inline src/command-parser/expand-npm-wildcard.js]: I rolled this line back to what it is on `master`, as some other information will exist on `commandInfo`

**Comment by gustavohenke** [comment]
Closing as it's fixed by #211.

**Pull request closed without merging**